flycheck.el

```diff
@@ -6262,6 +6263,63 @@ Requires DMD 2.066 or newer.  See URL `http://dlang.org/'."
          (one-or-more " ") (message) line-end))
   :modes d-mode)
 
+(defun flycheck-elixir--find-default-directory (_checker)
+  "Come up with a suitable default directory to run CHECKER in.
```

Comment:
As with the names, I'd prefer to keep the functions that can be reused checker-agnostic.